spec/proposals_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "Visit a proposal", type: :system, perform_enqueued: true do
+  let(:organization) { create :organization }
+  let(:participatory_process) { create :participatory_process, organization: organization }
+  let(:proposals_component) { create :component, manifest_name: :proposals, participatory_space: participatory_process }
+  let!(:proposal) { create :proposal, component: proposals_component }
+  let!(:awesome_config) { create :awesome_config, organization: organization, var: :use_markdown_editor, value: true }
+
+  before do
+    switch_to_host(organization.host)
+    # page.visit main_component_path(proposals_component)
+    # execute_script("window.DecidimAwesome.use_markdown_editor = true;")
```

Comment:
we can :fire: this, right?

Reply:
Change it, because in the next version of awesome won't work. This is just a test to ensure it is working